backend/Controllers/UserController.js

```diff
@@ -114,124 +120,43 @@ const verifyUserByToken = async (req, res) => {
-          <li>Name: ${name}</li>
-          <li>Email: ${email}</li>
-        </ul>
-        <h3>Message:</h3>
-        <p>${message}</p>
-
-        <h3>Rating:</h3>
-        <p style="font-size: 24px; color: #FFD700;">${filledStars}${emptyStars}</p> <!-- Display the stars -->
-      `, // HTML body
-    };
-
-    await transporter.sendMail(mailOptions);
+    // Save feedback to the database
+    await newfeedback.save();
 
     return res
       .status(200)
-      .json({ success: true, message: "Message sent successfully!" });
+      .json({
+        success: true,
+        message: "Feedback stored successfully!",
+        newfeedback,
+      });
   } catch (error) {
-    console.error("Error sending email:", error);
+    console.error("Error storing feedback:", error);
     return res
       .status(500)
-      .json({ success: false, message: "Error sending email" });
+      .json({ success: false, message: "Error storing feedback" });
   }
 }
-const forgotPassword = async function (req, res) {
-  const { email } = req.body;
-  const user = await User.findOne({ email });
-  try {
-    if (!user) {
-      return res
-        .status(404)
-        .json({ success: false, message: "Email doesnt exist" });
-    } else {
-      const token = jwt.sign({ userId: user._id }, process.env.SECRET, {
-        expiresIn: "5m",
-      });
-      var transporter = nodemailer.createTransport({
-        service: "gmail",
-        auth: {
-          user: process.env.SMTP_EMAIL,
-          pass: process.env.SMTP_PASSWORD,
-        },
-      });
-      var userFullName = user.firstName + " " + user.lastName;
-      var mailOptions = {
-        from: process.env.SMTP_EMAIL,
-        to: email,
-        subject: "Password Reset | TastyTrails",
-        html: `<p>Hi <b> ${userFullName},</b><br>Use the below link to reset you password. Remember, the link will expire in 10 minutes.<br> ${process.env.FRONT_END_URL}/reset_password/${token}`,
-      };
-
-      transporter.sendMail(mailOptions, function (error, info) {
-        if (error) {
-          return res
-            .status(404)
-            .json({ success: false, message: "Email not sent" });
-        } else {
-          return res
-            .status(200)
-            .json({ success: true, message: "Email sent succesfully" });
-        }
-      });
-    }
-  } catch (error) {
-    console.log(error);
-  }
-};
-
-const resetPassword = async function (req, res) {
-  const { token } = req.params;
-  const { password } = req.body;
-  try {
-    const decoded = jwt.verify(token, process.env.SECRET);
-    console.log(decoded);
-    const id = decoded.userId;
-    const hashPassword = await bcrypt.hash(password, 10);
-    await User.findByIdAndUpdate({ _id: id }, { password: hashPassword });
-    return res
-      .status(200)
-      .json({ success: true, message: "Password reset succesfully" });
-  } catch (error) {
-    return res.status(400).json({ success: false, message: "Invalid token" });
-  }
-};
 
```

Comment:
@haseebzaki-07 why the above functions are deleted ?? They are important

Reply:
@AlfiyaSiddique brought them back